packages/manager/modules/pci/src/projects/project/kubernetes/details/service/terminate/terminate.html

```diff
@@ -9,14 +9,11 @@
                data-on-dismiss="$ctrl.goBack()"
                data-loading="$ctrl.isDeleting"
                data-type="warning">
-        <p class="oui-paragraph" data-translate="kube_service_terminate_warning"></p>
-        <oui-field data-label="{{:: 'kube_service_terminate_enter' | translate }}">
-            <input class="oui-input"
-                   type="text"
-                   name="kubeClusterTerminateConfirm"
-                   data-ng-model="$ctrl.terminateConfirmation"
-                   data-ng-pattern="$ctrl.TERMINATE_INPUT"
-                   required>
-        </oui-field>
+        <p class="oui-paragraph"
+           data-translate="kube_service_terminate_description"
+           data-translate-values="{ clusterName: $ctrl.cluster.name }"></p>
+        <oui-message data-type="warning" data-dismissable="false">
```

Comment:
Same here : no need to set `dismissable`

Reply:
This is done :)